litmus-portal/frontend/src/views/AnalyticsDashboard/MonitoringDashboardPage/DataSourceInactiveModal/index.tsx

```diff
@@ -0,0 +1,100 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import { Typography } from '@material-ui/core';
```

Comment:
Remove the `disable lint statement` and do the appropriate changes

Reply:
Done